fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -365,6 +367,10 @@ private boolean isAuthorized(final Subject currentUser, final HttpServletRequest
                 }
             }
         case "POST":
+            if (isTxEndpoint && currentUser.isAuthenticated()) {
+                log.debug("POST allowed to transaction endpoint");
```

Comment:
Could you add the "currentUser"s ID to the debug message?

Reply:
Done.